modules/create-object-collections.adoc

```diff
@@ -0,0 +1,66 @@
+. Optional: Add additional collections to a collection. Grouping collections and adding them to other collections can be useful if you want to create small collections based on deployments, then reuse and combine those smaller collections into larger, hierarchical collections. To add additional collections to a new collection that you are creating:
+.. In the *Attached collections* section, select a collection from the *Available collections* list or enter text in the *Filter by name* field and press *->*. Clicking the name of the collection displays information about the collection, such as the name and rules for the collection. After viewing collection information, close the window to return to the *Available collections* list.
+.. Click *+Attach*. Collections you attach are listed in the *Attached collections* section.
+. Click *Save*. 
+
+[id="creating-collection-rules"]
+== Creating collection rules
+
+Use the following guidelines when creating collection rules:
+
+* You must configure at least one rule or attach another collection to the new one that you are creating. 
++
+[NOTE]
+====
+For {product-title-short} release 3.74, collections are supported only for deployments, so you must configure a deployment object.
+====
+*  You first configure rules to include all or specific deployments, and then you can further filter the collection to include objects meeting certain criteria in namespaces and clusters.
+
+.Procedure
+
+. In the *Deployments* section, choose one of the following options from the drop-down list:
+
+* *All deployments*: Includes all deployments in the collection. If you choose this option, you must then filter the collection using namespaces or clusters.
+* *Deployments with names matching* or *Deployments with labels matching*: Click one of these options to select by name or label and then click one of the following options:
+** Select *An exact value of* and enter the exact name of the object.
+** Select *A regex value of* to use regular expression, also called "regex," to search for an object. This option is useful if you do not know the exact name of the object. A regular expression is a string of letters, numbers, and symbols that defines a pattern. {product-title-short} uses this pattern to match characters or groups of characters and return results. For more information about regular expression, see "Regular-Expressions.info" in the "Additional resources" section.
+. Optional: To add more name matching criteria, click *OR* and configure another exact or regular expression value.
+
+The following example provides the steps for configuring a collection for a medical application. In this example, you want your collection to include the `reporting` deployment, a database called `patient-db`, and you know that the database is located in a namespace with a label of `medical`. You would perform the following steps:
```

Comment:
I love this idea of an example. However, I think we need to back this up with example yamls. Like where does `patient-db` come from? etc.


Here is a reference:
https://docs.openshift.com/container-platform/3.11/dev_guide/deployments/how_deployments_work.html#creating-a-deployment-configuration

Reply:
Still investigating this issue - in progress

Reply:
@md2119 - for now, would it be OK to leave this and flesh out a larger configuration example later, with yaml files? I have a larger configuration section as an action item, but am trying to get someone to sign on to help me provide that. Boaz said he would help, but I haven't had time to work on this yet.